schedule_lessons/dashboard/static/dashboard/js/settings.js

```diff
@@ -55,6 +55,15 @@ $(document).ready(() => {
       });
     }
   });
+
+  $(".preference").click((event) => {
+    const preference_id = event.target.id;
```

Comment:
since we are only using a few properties of this event object, let's try this:

```
const { id, checked } = event.target;
```

and then use `id` and `checked`